fieldservice_partner_relation_type/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1 @@
+* Steven Campbell <smangukiya@opensourceintegrators.com>
```

Comment:
smangukiya?

Reply:
@max3903 : Good catch, I have not noticed that :)

Reply:
Whoops! I used Sandip's README as a reference for mine